test/goodeggs_money.test.js

```diff
@@ -0,0 +1,345 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
this is because this repo is really old and using chai without dirty-chai, so you can do weird things like `expect(foo).to.be.defined` without calling the function.

just `yarn remove chai && yarn add --dev goodeggs-test-helpers` and `import {expect} from 'goodeggs-test-helpers'` instead, and then you can add the `()` to all these weird expect chains